tests/unit/bivariate/test_clayton.py

```diff
@@ -22,8 +22,8 @@ def setUp(self):
     def test_fit(self):
         """On fit, theta and tau attributes are set."""
         self.copula.fit(self.X)
-        self.assertAlmostEqual(self.copula.tau, 0.7877, places=3)
-        self.assertAlmostEqual(self.copula.theta, 7.4218, places=3)
+        assert self.copula.tau, 0.7877
```

Comment:
This change does not seem correct.
Here we are saying `assert self.copula.tau` and if it fails, give 0.7877 as the message.
I think we should rather be using [pytest.approx](https://docs.pytest.org/en/6.2.x/reference.html#pytest-approx)

Reply:
Oh, I forgot to write this test (that was the placeholder I left there), great catch! I'll use the same syntax I'm using in the other tests, for consistency (since you mentioned it's fine in the comment below).